server/server/migrations/migrations/add_po_htmls_collection_005.py

```diff
@@ -0,0 +1,16 @@
+from common.arangodb import get_db
+from migrations.base import Migration
+
+
+class SecondMigration(Migration):
+    migration_id = 'add_po_htmls_collection_005'
+    tasks = ['create_collection']
+
+    def create_collection(self):
+        db = get_db()
+
+        po_htmls = db.create_collection('po_htmls')
+        po_htmls.add_hash_index(fields=['uid'], unique=False)
```

Comment:
shouldn't the uid be unique?

Reply:
It is uid of sutta. we can have multiple translations of a single sutta.

Reply:
ah, I see.